app/scripts/directives/deleteLink.js

```diff
@@ -79,12 +79,9 @@ angular.module("openshiftConsole")
             group: 'autoscaling'
           }, hpa.metadata.name, { namespace: scope.projectName })
           .then(function() {
-            showAlert({
-              name: hpa.metadata.name,
-              data: {
+            NotificationsService.addNotification({
                 type: "success",
                 message: "Horizontal Pod Autoscaler " + hpa.metadata.name + " was marked for deletion."
```

Comment:
not new, but this casing isn't consistent with how we humanize resource types elsewhere. Should we change this one?